pkg/controller/appdefinition/testdata/cronjob/expected.yaml

```diff
@@ -47,6 +47,11 @@ spec:
           terminationGracePeriodSeconds: 5
           containers:
             - name: oneimage
+              resources:
```

Comment:
It seems strange to me that all of these tests would produce memory limits and requests of `"0"`. Can we leave out the memory requests and limits if they will output `"0"`?